### PR TITLE
Update ediscovery.md

### PR DESCRIPTION
**Pull request by msbemba**
Added M365 Business standard and Basic to content search after confirming here -https://aka.ms/M365BusinessPlans

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/7709

File: microsoft-365/compliance/ediscovery.md [hunk 1]
```diff
@@ -127,6 +127,8 @@ The following sections show the minimum subscription requirements for Content se
 - Microsoft 365 G1 subscription
 - Microsoft 365 F1 or F3 subscription, or F5 Security add-on
 - Microsoft 365 Business Premium subscription
+- Microsoft 365 Business Standard
+- Microsoft 365 Business Basic
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/ediscovery.md]:
```suggestion
- Microsoft 365 Business Standard subscription
- Microsoft 365 Business Basic subscription
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@markjjo Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@markjjo Gentle reminder.

**Comment by yogkumgit** [comment]
@markjjo Kindly merge this PR. Thanks

**Pull request merged**